eth/transaction_from_raw_test.go

```diff
@@ -29,6 +31,32 @@ func TestTransaction_FromRaw(t *testing.T) {
 	require.True(t, tx.IsProtected())
 }
 
+// polygon
```

Comment:
Hah, didn't mean to stage this, it's an unrelated test to make sure that we can parse Polygon txs correctly, though at this point I don't see any harm in leaving it in.